src/main/scala/cromwell/engine/db/slick/SymbolComponent.scala

```diff
@@ -93,4 +96,12 @@ trait SymbolComponent {
       workflowExecution <- symbol.workflowExecution
       if workflowExecution.workflowExecutionUuid === workflowExecutionUuid
     } yield symbol)
+
+  val symbolsForWorkflowOutput = Compiled(
+    (workflowExecutionUuid: Rep[String]) => for {
+      symbol <- symbols
+      if symbol.index === -1 && symbol.reportableResult === true
```

Comment:
is it possible that something with an index != -1 could be set as a reportable result?